examples/component-testing/create-react-app/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
create-react-app folder generated using `npx create-react-app create-react-app --template typescript`. I just install few modules and configure testplane and vite